src/style.css

```diff
@@ -1,3 +1,71 @@
+ html {
+  --blue-colour: #00ACDC;
+
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  background: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo__link {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  margin-right: 50px;
```

Comment:
it's better to add inline-padding to the `header` in such cases - that's not the margin of `nav` or `logo` - definitely looks like internal indent of the padding, no matter of content